reactive/kubernetes_master.py

```diff
@@ -3725,4 +3725,5 @@ def configure_default_cni():
         "run": start_master,
         "clear_flags": ["kubernetes-master.components.started"],
     },
+    "kubelet": {"run": reconfigure_kubelet, "clear_flags": []},
```

Comment:
@Cynerva is this the right function to run to reconfigure the kubelet

https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/207/files#diff-b3099d2217027f2f460740149252f0dca742dbb5deeb274bc17559fb648b1d86R3685-R3692

Reply:
it seemed appropriate but I wasn't totally sure

Reply:
:+1: Looks reasonable to me